udm-le.env

```diff
@@ -147,7 +147,7 @@ LEGO_PATH="${UDM_LE_PATH}/.lego"
 # These should only change if Unifi-OS core changes require it
 CERT_IMPORT_CMD="java -jar /usr/lib/unifi/lib/ace.jar import_key_cert"
 UBIOS_CONTROLLER_CERT_PATH="/data/unifi-core/config"
-UBIOS_RADIUS_CERT_PATH="/data/udapi-config/raddb/certs"
+UBIOS_RADIUS_CERT_PATH="/etc/freeradius/3.0/certs/"
```

Comment:
Nix the trailing / here, it's accounted for later anyway.

Reply:
Yep. Removed.